lib/galaxy/webapps/galaxy/api/library_datasets.py

```diff
@@ -22,7 +22,8 @@
     folders,
     lddas,
     library_datasets,
-    roles
+    roles,
+    hdas
```

Comment:
This import should go after `folders,` above.

Reply:
![img_20190108_135406896](https://user-images.githubusercontent.com/1814954/50852408-705c0880-134d-11e9-8eb0-8b289eb245cb.jpg)